front/app/modules/commercial/representativeness/admin/hooks/createRefDataSubscription.ts

```diff
@@ -1,11 +1,4 @@
 // services
-import {
```

Comment:
Let's rename this file to `parseReferenceData` or something, since it's not doing anything with subscriptions anymore? Could be refactored to be more functional too instead of having this weird `setter` setup, but I'll leave that up to you

Reply:
Agree to rename but I'm not going to refactor the mechanism that is in place. It's not ideal but a bit out of scope for me here.